### ISSUE TITLE
[1.21.1] Incompatible with VanillaTweaks DataPack Graves

### ISSUE THREAD

**Issue by Nova-Katt** [wisp-forest/accessories#131]
Minecraft: 1.21
Fabric: 1.16.5
Accessories: 1.0.0 Beta 35 for 1.21
And then: [Gameplay Changes - **Graves**](https://vanillatweaks.net/picker/datapacks/)

We are running these two (amonst other mods) and have noticed that the Graves datapack from VanillaTweaks doesn't work together with Accessories.
When you die your entire inventory is saved in a gravestone. When you run back to your death location and right click that grave you get all your stuff back, except the accessories. They seem to be lost.

I will post this same issue to the VanillaTweaks team aswell since I'm unsire if the incompatibility is fixable on your side or their side. :)

**Comment by Nova-Katt** [comment]
Oh, as addition. We used Trinkets instead of Accessories before, but changed due to the Things mod switching to acceessories.
Trinkets was compatible. Accessories seems to not be.

**Comment by Dragon-Seeker** [comment]
Odd to say the least that this dose not work though I have little idea exactly how they captured the trinkets stacks so Idk if they had compat for such or if it something to deal with accessories but I recently changed how stacks are dropped to use vanilla methods more so it may work better with the recent 1.1.0 version

**Comment by Nova-Katt** [comment]
Yeah it's weird indeed. I've been thinking of seeing if there simply is another actual grave mod instead, that has compat with accessories, rather than this datapack, but we'll see.
If you manade to find some fix before I find another mod then woohoo! :P

**Comment by Noaaan** [comment]
Could you at least do as Blod mentioned and test this with Accessories 1.1.0?

**Comment by Dragon-Seeker** [comment]
AS noaaan stated you should give it shot but YIGD or Universal Graves have full support with the latter being a polymer mod so its fully server-side.

**Comment by Nova-Katt** [comment]
I tried both Beta4 and Beta6 but they both crashed on me.
So I'm back to Beta35 which actually let's me start the game. :P

Here are the crash logs for both Beta4 and Beta6
[crash-2024-10-19_13.59.05-client.txt](https://github.com/user-attachments/files/17445016/crash-2024-10-19_13.59.05-client.txt)
[crash-2024-10-19_14.01.04-client.txt](https://github.com/user-attachments/files/17445017/crash-2024-10-19_14.01.04-client.txt)

**Comment by Dragon-Seeker** [comment]
You need to update the Trinkets Compat layer when updating to 1.1.0

**Comment by Nova-Katt** [comment]
> You need to update the Trinkets Compat layer when updating to 1.1.0

3.10.0+beta.11 (the one i have) is the latest one available for 1.21.
The later versions are for 1.21.1 which we don't run yet (since we're waiting for several other mods to port over to .1

So no compat to update. ^^;

**Comment by Noaaan** [comment]
I heavily suggest that you should just update. Version 1.21.1 hotfixed a single crash in vanilla Minecraft, and almost no mods were affected (meaning they will work on that version)

**Comment by Nova-Katt** [comment]
I see. I thought there were more stuff affected. I'll give it a try later then. Too tired to fiddle with that right now, but yeah I'll give it a go updating to .1 later. I'm running like 160+ mods so it's a bit of a hassle (and probably a lot of "it works but the logs does have errors" because of so many mods. xD

**Comment by Dragon-Seeker** [comment]
@Nova-Katt Any update to if this was fixed or not?

**Comment by GrantGryczan** [comment]
Vanilla Tweaks data pack developer here--this incompatibility is caused by #261, which I just reported.

(But also note that even if this is fixed, you still have to enable Compatibility Mode in the interactive Graves config (`/function graves:config`) for Graves to recognize items in modded slots.)

**Comment by Dragon-Seeker** [comment]
Should be fixed for 1.21.1.
